Makefile

```diff
@@ -0,0 +1,18 @@
+.PHONY: all
+all: prepare lint test_rules
```

Comment:
Don't you want to run `pint` as well ?

Reply:
I ran it locally and noticed it reinstall pint every time. It also not on gitignore so having it here increases the risk of it being pushed to git. I opened a ticket to address those. Once addressed, we should add it as well here.

BTW, target `all` is only run in local dev envs. on CI, the targets are run independently according to the ci step, so pint is running there.